### ISSUE TITLE
Improve fonts

### ISSUE THREAD

**Issue by mrmartineau** [refined-bitbucket/refined-bitbucket#35]
@andremw I think improving the fonts on Bitbucket would also make a great addition to this extension. 

Adding a CSS file that contains the below code would significantly improve the interface.

```css
body {
  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen, Ubuntu, Cantarell, "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif !important;
}
```

Let me know what you think and I can submit a PR for it.

**Comment by andremw** [comment]
Hi, @mrmartineau! Thanks for the suggestion. I tried it and I think it gets really nice. Feel free to go ahead and submit a PR!

**Comment by reyronald** [comment]
Closed via #126
